addon/src/decorators/key-responder.js

```diff
@@ -1,4 +1,4 @@
-import { inject as service } from '@ember/service';
+import { service } from '@ember/service';
 import { registerDestructor } from '@ember/destroyable';
 
```

Comment:
Hello 👋 

Thanks for the the fix. We have the same problem in our Ember project (6.3) 👏 

On the [deprecation page](https://deprecations.emberjs.com/id/importing-inject-from-ember-service/), it says : 

> If you're working on a library that needs to support ember-source prior to 4.1, you can support both styles of service via:
> import * as emberService from '@ember/service';
> const service = emberService.service ?? emberService.inject;

I think you need to do the same thing due to this library supporting Ember 3.28 : 

```suggestion
import { service } from '@ember/service';
import { registerDestructor } from '@ember/destroyable';

const service = emberService.service ?? emberService.inject;
```